valheim-bootstrap

```diff
@@ -1,5 +1,15 @@
 #!/bin/bash
 RESTART_CRON=${RESTART_CRON-0 5 * * *}
+TZ=${TZ:-Etc/UTC}
```

Comment:
Same comment as above, this file shouldn't be part of this PR?! Did you git rebase on current main?